docs/websites/nginx/nginx-and-pagespeed-on-ubuntu-14-04.md

```diff
@@ -0,0 +1,132 @@
+---
+author:
+    name: Linode Community
+    email: docs@linode.com
+description: 'Learn how you can install Nginx with the ngx_pagespeed module on Ubuntu 14.04.'
+keywords: 'install ngx_pagespeed, install Nginx from source'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'Friday, September 11th, 2015'
+modified: Friday, September 11th, 2015
+modified_by:
+    name: Linode
+title: 'How to Install ngx_pagespeed module with Nginx on Ubuntu 14.04'
+contributor:
+    name: Vaibhav Rajput
+    link: https://twitter.com/IamVRajput
+external_resources:
+ - '[Google PageSpeed Filter Docs](https://developers.google.com/speed/pagespeed/module/config_filters)'
+---
+
+This document describes how you can install Nginx (pronounced engine-x) with the ngx_pagespeed module. PageSpeed module helps us to minify CSS, javascript, images, remove whitespaces from HTML and many other features.
+
+Make sure that before starting this guide you have read through and completed our [Getting Started](/docs/getting-started/) guide.
+
+##Set the hostname
+
+Before you install any packages, ensure that your hostname is correct by completing the [Setting Your Hostname](/docs/getting-started#sph_setting-the-hostname) section of the Getting Started guide. Issue the following commands to verify:
+
+    hostname
+    hostname -f
+
+##System Setup
+
+Make sure your system is up to date using apt:
+
+    sudo apt-get update
+
+This ensures that all software is up to date and running the latest version.
+
+##Installing required packages
+
+Pagespeed module requires some extra packages which you should install to make it run properly, issue the following command in the terminal.
+
+    sudo apt-get install build-essential zlib1g-dev libpcre3 libpcre3-dev unzip

```

Comment:
For many of the commands, I see you've prefixed the commands with `sudo`. The guide currently has the user running as the root user. You can stick with the root user and drop the `sudo` prefix from all of these commands or you can walk them through setting up a normal user, logging in with that user, and then leave `sudo` on these commands.